packages/neo-one-build-tests/jest/base.js

```diff
@@ -32,7 +32,7 @@ module.exports = ({ path }) => ({
     `${jest_root}/serializers/buffer.js`,
   ],
   testPathIgnorePatterns,
-  transformIgnorePatterns: ['node_modules', '/packages/.*/lib/'],
+  transformIgnorePatterns: ['node_modules'],
```

Comment:
Is there a reason we don't need to include `lib` in this? I'm sure there is, I'm just curious